### DISCUSSION TITLE
Dell iDrac9, SuperMicro BMC IPMI, and Linux Cockpit VNC don't work with nginx-proxy-manager.

### DISCUSSION THREAD

**Discussion by LittleNewton** [NginxProxyManager/nginx-proxy-manager#2609] [Q&A]
Hello.

I have set a proxy for **Dell server**'s iDrac9 service. But when I opened the URL, an `ERROR 400` appeared.

```
Bad Request
Your browser sent a request that this server could not understand.
Additionally, a 400 Bad Request error was encountered while trying to use an ErrorDocument to handle the request.
```

Another problem concerns **Supermicro's BMC interface**. With nginx-proxy-manager, I can open it in the browser. However, it interacts very slowly. However, when I access it with its IP address, the speed is acceptable. Moreover, the VNC window of Supermicro BMC cannot be loaded under nginx-proxy-manager.

Last, **Windows Server's Admin Center** also cannot be opened under the proxy of nginx-proxy-manager.

- Dell iDrac - **ERROR 400**
- SuperMicro BMC - **Slow** and **no VNC**
- Windows Admin Center - **Cannot open**

I appreciate any helpful information.

**Comment by cryptedx** [comment]
I 2nd this. My Supermicro BMC is slow and it does not load the saved settings. Not usable at all. With IP everything works smooth.
<img width="1495" alt="image" src="https://github.com/NginxProxyManager/nginx-proxy-manager/assets/20707078/e747cbd2-f204-4d78-8ae5-935bfddc8a13">

**Comment by peterge-misoft** [comment]
We are facing the same problem with supermicro ipmi...
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/162673647/4975eebc-d40e-4a0c-b9b9-64497db6a979)
v2.10.4 © 2023

**Reply by peterge-misoft** [reply]
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/162673647/1655a5fb-c900-4a7f-b0e0-6e6922eca2d3)
Same with v2.11.1 © 2024

**Comment by Skylinar** [comment]
Is there a solution for it? I'm having the same issue with supermicro BMC.

**Reply by peterge-misoft** [reply]
my "solution" is to not use npm bor bmc websites and to access it via the ip directy...
